pkg/proxy/apis/config/v1alpha1/zz_generated.conversion.go

```diff
@@ -123,6 +123,7 @@ func autoConvert_v1alpha1_KubeProxyConfiguration_To_config_KubeProxyConfiguratio
 	}
 	out.ShowHiddenMetricsForVersion = in.ShowHiddenMetricsForVersion
 	out.DetectLocalMode = config.LocalMode(in.DetectLocalMode)
+	out.CleanupAndExit = in.CleanupAndExit
```

Comment:
The round trip test is failing because you are missing this line for the reverse ```autoConvert_config_KubeProxyConfiguration_To_v1alpha1_KubeProxyConfiguration``` on line 167.

You can add this automatically by either running the test `bazel test //pkg/proxy/apis/config/scheme:go_default_test` or running `make update`